bundle/src/main/java/com/adobe/acs/commons/workflow/bulk/execution/impl/runners/AEMTransientWorkflowRunnerImpl.java

```diff
@@ -212,8 +213,13 @@ public void run() {
                     workspace.commit();
                 }
             } catch (Exception e) {
-                log.error("Error processing periodic execution: {}", e);
+                log.error("Error processing periodic execution for job [ {} ] for workspace [ {} ]", new String[]{ jobName, workspace.getPath() }, e);
```

Comment:
The String array is not needed here `log.error("Error processing periodic execution for job [ {} ] for workspace [ {} ]", jobName, workspace.getPath(), e);` should just work fine.